<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QDialog" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADMIN</string>
  </property>
  <widget class="QPushButton" name="admAmbilData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ambil Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admHapus">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admTambah">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Alamat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nomor Hp</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
